src/features/ITGovernance/Admin/GRBReview/GRBReviewForm/Presentation/index.tsx

```diff
@@ -1,60 +1,512 @@
-            <FormGroup error={!!error}>
-              <Label
-                htmlFor={field.name}
-                className="text-normal"
-                requiredMarker
-              >
-                GRB Date
-              </Label>
-
-              <DatePicker
-                {...field}
-                id={field.name}
-                defaultValue={field.value}
-              />
-            </FormGroup>
-          )}
-        />
-      </GRBReviewFormStepWrapper>
-    </EasiFormProvider>
+    <>
+      {grbReview.grbReviewType === SystemIntakeGRBReviewType.STANDARD ? (
+        <EasiFormProvider<StandardPresentationFields> {...standardForm}>
+          <GRBReviewFormStepWrapper
+            grbReview={grbReview}
+            onSubmit={onSubmitStandard}
+            disabled={!watch('grbMeetingDate.grbDate')}
+          >
+            <Fieldset>
+              <Grid desktop={{ col: 6 }}>
+                <div className="border-top-1px border-base-lighter margin-top-3 easi-text-normal">
+                  <h4 className="margin-top-1 margin-bottom-05">
+                    {t('presentationGRBReviewForm.heading')}
+                  </h4>
+
+                  <p className="margin-y-0 text-base">
+                    {t('presentationGRBReviewForm.description')}
+                  </p>
+
+                  <Alert type="info" slim className="margin-top-105">
+                    {t('presentationGRBReviewForm.alert')}
+                  </Alert>
+
+                  <Controller
+                    name="grbMeetingDate.grbDate"
+                    control={control}
+                    rules={{
+                      required: 'presentationGRBReviewForm.required'
+                    }}
+                    render={({
+                      field: { ref, ...field },
+                      fieldState: { error }
+                    }) => (
+                      <FormGroup error={!!error}>
+                        <Label
+                          htmlFor={field.name}
+                          className="text-normal"
+                          requiredMarker
+                        >
+                          {t('presentationGRBReviewForm.meetingDateLabel')}
+                        </Label>
+
+                        <HelpText className="margin-top-1">
+                          {t(
+                            'presentationGRBReviewForm.meetingDateDescription'
+                          )}
+                        </HelpText>
+
+                        {!!error?.message && (
+                          <FieldErrorMsg>{t(error.message)}</FieldErrorMsg>
+                        )}
+
+                        <DatePickerFormatted
+                          {...field}
+                          id={field.name}
+                          defaultValue={grbReview.grbDate || ''}
+                          onChange={e => field.onChange(e || undefined)}
+                          dateInPastWarning
+                        />
```

Comment:
The `DatePickerFormatted` component is causing the field to always be dirty because the `onChange` runs on render. The `defaultValue` for this field is a different format than the value returned from the `onChange` function, so it's registering as dirty as soon as that function executes and updates the field value.

Functionally the form still works so I don't think this necessarily has to be fixed for this PR to go through, but I think it's something we need to address at some point.